### PR TITLE
fix(List/Toolbar): comply with UI guidelines

### PR DESCRIPTION
**Pull request by mmaillart**
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)
some toolbar controls styles were incorrect (labels, dropdown, etc.)

**What is the new behavior?**
UI Styles complies with UI guidelines

**Does this PR introduce a breaking change?**

- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

screen shot 2016-11-28 at 17 40 57

### CODE REVIEW

File: src/List/Toolbar/SelectSortBy/SelectSortBy.component.js [hunk 1]
```diff
@@ -58,12 +61,11 @@ function SelectSortBy(props) {
 			</NavDropdown>
 			<Button
 				id={props.id && `${props.id}-sort-order`}
-				className="navbar-btn"
+				className="navbar-btn btn"
 				bsStyle="link"
 				role="button"
 				onClick={toggleSortOrder}
 			>
-				<Icon name={props.sortDesc ? 'fa-sort-desc' : 'fa-sort-asc'} />
```
Comment (jmfrancois) [inline src/List/Toolbar/SelectSortBy/SelectSortBy.component.js]: On the spec there are icons why did you remove them ?
http://guidelines.talend.com/document/92132#/navigation-layout/list

File: src/List/Toolbar/SelectSortBy/SelectSortBy.scss [hunk 2]
```diff
@@ -1 +1,12 @@
 // SelectSortBy styles
+
+$tc-list-toolbar-display-color: $dove-gray !default;
+
+.tc-list-toolbar-sort-by {
+	:global(.navbar-btn.btn) {
+		color: $tc-list-toolbar-display-color;
+		font-size: 15px;
+		padding-top: 6px;
```
Comment (jmfrancois) [inline src/List/Toolbar/SelectSortBy/SelectSortBy.scss]: those should be defined variables from the guidelines

Comment (jsomsanith) [inline src/List/Toolbar/SelectSortBy/SelectSortBy.scss]: +1
And if we remove the actions buttons (like in the guideline) this button is bigger than the toolbar height, it pushes it. The margin + padding should be reworked

File: src/List/Toolbar/Toolbar.scss [hunk 3]
```diff
@@ -1 +1,10 @@
 // Toolbar styles
+$tc-list-toolbar-display-color: $silver !default;
```
Comment (jsomsanith) [inline src/List/Toolbar/Toolbar.scss]: Is it really readable ?

File: src/List/Toolbar/SelectSortBy/SelectSortBy.component.js (hunk 1 above)
Comment (mmaillart) [inline src/List/Toolbar/SelectSortBy/SelectSortBy.component.js]: this was not an icon but a dropdown (Ascending, Descending, None) list, same as for the "Sort by" dropdown. The fact that the sense of the order changes on_click, we don't need anymore to have this as a dropdown.

File: src/List/Toolbar/SelectSortBy/SelectSortBy.scss (hunk 2 above)
Comment (mmaillart) [inline src/List/Toolbar/SelectSortBy/SelectSortBy.scss]: this is for the padding-top ?

File: src/List/Toolbar/Toolbar.scss (hunk 3 above)
Comment (mmaillart) [inline src/List/Toolbar/Toolbar.scss]: on my display yes. it is too bright on yours ?

Comment (jsomsanith) [inline src/List/Toolbar/Toolbar.scss]: On mine the contrast with its background is not really good
